hummingbot/client/command/config_command.py

```diff
@@ -114,6 +114,9 @@ def get_all_available_config_keys() -> List[str]:
         self.placeholder_mode = False
 
         if choice.lower() in {"y", "yes"}:
+            # Clear application states that are specific to config
+            self.starting_balances = {}
```

Comment:
I think this line belongs in `self._stop_loop`

Reply:
If we reset the starting balance in stop loop, we will lose starting balance data when we start and stop strategies.
We only want to reset starting balance when we reconfig markets/strategies.